app/Http/Controllers/UserController.php

```diff
@@ -85,16 +88,78 @@ public function destroy($id)
             } else {
                 $userName = $user->name;
                 $user->delete();
-
                 return redirect()->route('user.index')
                                  ->withMessage($userName.trans('users.delete.delete_successful'));
             }
         } catch (Exception $modelNotFound) {
+            return redirect()->route('user.index')->withErrors(trans('users.error_message'));
         }
-
         return redirect()->route('user.index')->withErrors($errors);
     }
-
+    
+    /**
+     * Update user infomation
+     *
+     * @param Request $request hold all data from request
+     * @param integer $id      determine specific user
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function update(UserRequest $request, $id)
+    {
+        try {
+            $input = $request->all();
+            $user = User::findOrFail($id);
+            unset($input['birthday']);
+            $user->fill($input);
+            $user->birthday = date('Y-m-d', strtotime($request->birthday));
```

Comment:
I knew this is convert to MySQL datetime, but I think it's better if we define it as global too.